pkg/logout/logout_all.go

```diff
@@ -0,0 +1,21 @@
+package logout
```

Comment:
I don't think this needs to be its own file. You could put this in the `logout.go` file and `All` will be exported as normal.

Reply:
Ok sounds good I'll move it into `logout.go`